src/components/molecules/Wallet/Details.tsx

```diff
@@ -26,6 +27,39 @@ export default function Details(): ReactElement {
     setProviderInfo(providerInfo)
   }, [web3Provider])
 
+  async function addOceanToWallet() {
+    const tokenMetadata = {
+      type: 'ERC20',
+      options: {
+        address: config.oceanTokenAddress,
+        symbol: networkId === 137 ? 'mOCEAN' : 'OCEAN',
+        decimals: 18,
+        image:
+          'https://raw.githubusercontent.com/oceanprotocol/art/main/logo/token.png'
+      }
+    }
+    web3Provider.sendAsync(
+      {
+        method: 'wallet_watchAsset',
+        params: tokenMetadata,
+        id: Math.round(Math.random() * 100000)
+      },
+      (err: string, added: any) => {
+        if (err || 'error' in added) {
+          Logger.log(
```

Comment:
nice, I like the more user-focused error messaging, has small typo `Counld't` at beginning. For errors we can then really use `Logger.error()` instead of `.log()`